es6.html

```diff
@@ -0,0 +1,116 @@
+    <li>Finally, all <strong>in progress</strong> features can be activated individually by their respective harmony flag (e.g. <code>--harmony_arrow_functions</code>), although this is highly discouraged unless for testing purposes.</li>
+    </ul>
+  </div>
+
+      <h2 class="faq-title">
+         Which ES6 features ship with io.js by default (no runtime flag required)?
+      </h2>
+      <div class="faq-body">
+      <ul class="task-list">
+        <li>Block scoping</li>
+        <ul class="task-list">
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let">let</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const">const</a></li>
+          <li><code>function</code>-in-blocks (<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode">strict mode only)</a></li>
+        </ul>
+        <li>Collections</li>
+        <ul class="task-list">
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map">Map</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap">WeakMap</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set">Set</a></li>
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet">WeakSet</a></li>
+        </ul>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*">Generators</a></li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Numeric_literals">Binary and Octal literals</a></li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise">Promises</a></li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla#Additions_to_the_String_object">New String methods</a></li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol">Symbols</a></li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings">Template strings</a></li>
+      </ul>
+
+      <p>You can view a more detailed list, including a comparison with other engines, on the <a href="https://kangax.github.io/compat-table/es6/">compat-table</a> project page.</p>
+    </div>
+    </div>
+
+    <div class="faq-item">
+      <h2 class="faq-title">
+         Which ES6 features are behind the --es_staging flag?
+      </h2>
+      <div class="faq-body">
+        <ul class="task-list">
```

Comment:
I'd add a note concerning the recents commits which disable ES6 Classes and Object Literal Extensions.

```html
Note: technically, V8 3.31.71.4 ships with ES6 Classes and Object Literal Extensions but due to some problematic issues with the specification, it has been <a href="https://groups.google.com/forum/#!msg/v8-users/To0IehKbwH8/CA9fWRsy7nEJ">unshipped from later versions of V8</a>. To avoid breaking changes after the initial release of io.js, they have been manually disabled for now (see commits <a href="https://github.com/iojs/io.js/commit/4e58211bb7f638b689e5e8e407b70b3c29be952f" target="_blank">4e58211bb7</a> and <a href="https://github.com/iojs/io.js/commit/a2751e3e1e70ecc9147f6dd6a58dbe9dc0d6b3be" target="_blank">a2751e3e1e</a>) but are expected to be reinstated in a future release of io.js.
```